### ISSUE TITLE
Ubuntu 16.04

### ISSUE THREAD

**Issue by DaniloCaporale** [robotology-playground/idynutils#7]
Hi all.
I can't get to compile idynutils in the superbuild on ubuntu 16.04, with ros-kinetic installed.
I have the following error:
```
danilo@walkmanbox:~/projects/robotology-superbuild/build/external/idynutils$ make
[ 34%] Built target idynutils
[ 42%] Built target _pydynutils
[ 42%] Built target copy_robot_data_files
[ 63%] Built target GTest-ext
Scanning dependencies of target RobotUtilsTest
[ 65%] Linking CXX executable ../bin/RobotUtilsTest
/usr/bin/ld: cannot find -loctomap
/usr/bin/ld: cannot find -loctomath
collect2: error: ld returned 1 exit status
tests/CMakeFiles/RobotUtilsTest.dir/build.make:507: recipe for target 'bin/RobotUtilsTest' failed
make[2]: *** [bin/RobotUtilsTest] Error 1
CMakeFiles/Makefile2:2319: recipe for target 'tests/CMakeFiles/RobotUtilsTest.dir/all' failed
make[1]: *** [tests/CMakeFiles/RobotUtilsTest.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
danilo@walkmanbox:~/projects/robotology-superbuild/build/external/idynutils$ 
```

I installed ros-kinetic-octomap* but does not solve the issue.

Thanks for you support.

Danilo

**Comment by DaniloCaporale** [comment]
Note: this happens on master branch updated to latest commit 
commit ba98bc361f173f7b063f6b6d1c9597fc578f7c9a
Author: Alessio Rocchi <alessio.rocchi@iit.it>
Date:   Tue Feb 21 13:35:51 2017 +0100

    Partially revert 859a863b55bb0

Thanks!

Danilo

**Comment by arocchi** [comment]
I attempted a fix at 43d5130e1f38918b16a6a6fe39d5a00fb636bd8f - could you take a look?
Soon we will have CI on again, I hope :+1:

**Comment by DaniloCaporale** [comment]
It works, now RobotUtilsTest links /opt/ros/kinetic/lib/liboctomap.so.1.8 and installing liboctomap-dev should not be required anymore.
Thanks man!
